docs/portal/transports/edit-transport.md

```diff
@@ -1,11 +1,11 @@
 # Edit a Transport
 
-To edit a transport, search the transport you want to edit in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Edit".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of the HDT portal.
+To edit a transport, click the "Action" button of the transport entry in the list and select "Edit". If the transport is not in the current page, you can search for it in the search box.
 
 ![null](</docs/resources/images/transports/edit-transport-1.png>)
 
-Edit the items to make the changes you want to make, then click the "Next" button in the top right corner of the page.
+Edit the items as desired, then click the "Next" button in the top right corner of the page.
```

Comment:
"as desired" is unnecessary.   The customer is always going to do what they want or need. I'd simplify to:  "Edit the settings and then click the "Next" button in the top right corner of the page."